test/e2e/main_test.go

```diff
@@ -137,7 +147,8 @@ func TestExternalService(t *testing.T) {
 			Name: "cache-infinispan-0",
 		},
 		Spec: ispnv1.InfinispanSpec{
-			Size: 1,
+			Size:  1,
```

Comment:
We should avoid cosmetic changes like this to align things. Did we create this or does `fmt` enforce it?

Reply:
yeah fmt wants it